entity-framework/core/what-is-new/ef-core-8.0/whatsnew.md

```diff
@@ -2237,3 +2235,353 @@ In EF Core 8, we now use the data format and column type name in addition to the
+    WHERE [s].[Region] = N'France'
+) AS [t] ON [c].[Id] = [t].[Id]
+```
+
+As a final example, in EF8, `ExecuteUpdate` can be used to update entities in a TPT hierarchy as long as all updated properties are mapped to the same table. For example, consider these entity types mapped using TPT:
+
+<!--
+    [Table("TptSpecialCustomers")]
+    public class SpecialCustomerTpt : CustomerTpt
+    {
+        public string? Note { get; set; }
+    }
+
+    [Table("TptCustomers")]
+    public class CustomerTpt
+    {
+        public int Id { get; set; }
+        public required string Name { get; set; }
+    }
+-->
+[!code-csharp[CustomerTpt](../../../../samples/core/Miscellaneous/NewInEFCore8/ExecuteUpdateDeleteSample.cs?name=CustomerTpt)]
+
+With EF8, the `Note` property can be updated:
+
+<!--
+        await context.TptSpecialCustomers
+            .Where(e => e.Name == name)
+            .ExecuteUpdateAsync(s => s.SetProperty(b => b.Note, "Noted"));
+-->
+[!code-csharp[TptUpdateNote](../../../../samples/core/Miscellaneous/NewInEFCore8/ExecuteUpdateDeleteSample.cs?name=TptUpdateNote)]
+
+Or the `Name` property can be updated:
+
+<!--
+        await context.TptSpecialCustomers
+            .Where(e => e.Name == name)
+            .ExecuteUpdateAsync(s => s.SetProperty(b => b.Name, b => b.Name + " (Noted)"));
+-->
+[!code-csharp[TptUpdateName](../../../../samples/core/Miscellaneous/NewInEFCore8/ExecuteUpdateDeleteSample.cs?name=TptUpdateName)]
+
+However, EF8 fails attempting to update both the `Name` and the `Note` properties because they are mapped to different tables. For example:
+
+<!--
+            await context.TptSpecialCustomers
+                .Where(e => e.Name == name)
+                .ExecuteUpdateAsync(s => s.SetProperty(b => b.Note, "Noted")
+                    .SetProperty(b => b.Name, b => b.Name + " (Noted)"));
+-->
+[!code-csharp[TptUpdateBoth](../../../../samples/core/Miscellaneous/NewInEFCore8/ExecuteUpdateDeleteSample.cs?name=TptUpdateBoth)]
+
+Throws the following exception:
+
+```text
+The LINQ expression 'DbSet<SpecialCustomerTpt>()
+    .Where(s => s.Name == __name_0)
+    .ExecuteUpdate(s => s.SetProperty<string>(
+        propertyExpression: b => b.Note,
+        valueExpression: "Noted").SetProperty<string>(
+        propertyExpression: b => b.Name,
+        valueExpression: b => b.Name + " (Noted)"))' could not be translated. Additional information: Multiple 'SetProperty' invocations refer to different tables ('b => b.Note' and 'b => b.Name'). A single 'ExecuteUpdate' call can only update the columns of a single table. See https://go.microsoft.com/fwlink/?linkid=2101038 for more information.
+```
+
+## Better use of `IN` queries
+
+When the Contains operator is used with a subquery, EF Core now generates better queries using SQL `IN` instead of `EXISTS`; aside from producing more readable SQL, in some cases this can result in dramatically faster queries. For example, consider the following LINQ query:
```

Comment:
```suggestion
When the `Contains` LINQ operator is used with a subquery, EF Core now generates better queries using SQL `IN` instead of `EXISTS`; aside from producing more readable SQL, in some cases this can result in dramatically faster queries. For example, consider the following LINQ query:
```